articles/azure-stack/azure-stack-sql-resource-provider-deploy.md

```diff
@@ -101,11 +101,22 @@ Change the default account information and passwords as needed for your Azure St
 
 
 ```powershell
-# Install the AzureRM.Bootstrapper module, set the profile and install the AzureStack module
+# Azure Stack version 1901 or later, only install the two modules below:

```

Comment:
we should not edit the SQL file, please revert the changes in this file. SQL RP does not support the 2.4.0 version yet.. they are planning to do another release, not sure when..  they still need the old modules